### PR TITLE
Switch to rails 6

### LINKED ISSUE #21: Switch to Rails 6
https://edgeguides.rubyonrails.org/6_0_release_notes.html

### PR DESCRIPTION
**Pull request by ag4ta**
Closes #21 
Fixes also nokogiri security alert

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,98 +1,71 @@
 GIT
-  remote: https://github.com/rails/rails.git
-  revision: d6eeab5265529746bb008150e08b50e4cc491591
-  branch: 6-0-stable
+  remote: https://github.com/rails/web-console.git
+  revision: 605351a8afa3b618dd5e50728d312ea3f1269f47
```
Comment (mpapis) [inline Gemfile.lock]: Go Rails Go ... stable version using git gem ;)

File: bin/setup [hunk 2]
```diff
@@ -9,24 +8,25 @@ def system!(*args)
...
+  # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
   # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  # Install JavaScript dependencies if using Yarn
+  # Install JavaScript dependencies
   # system('bin/yarn')
```
Comment (mpapis) [inline bin/setup]: Oh, actually should we uncomment it?

File: config/environments/development.rb [hunk 3]
```diff
@@ -29,16 +30,14 @@
...
-  # Store uploaded files on the local file system (see config/storage.yml for options)
+  # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
 
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (mpapis) [inline config/environments/development.rb]: Wow, this was a standard, did they finally made it default?

File: config/environments/production.rb [hunk 4]
```diff
@@ -31,16 +31,16 @@
...
 
-  # Mount Action Cable outside main process or domain
+  # Mount Action Cable outside main process or domain.
   # config.action_cable.mount_path = nil
   # config.action_cable.url = 'wss://example.com/cable'
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```
Comment (mpapis) [inline config/environments/production.rb]: :heart:

File: bin/setup (hunk 2 above)
Comment (ag4ta) [reply bin/setup]: I think yes, we should :D

File: config/environments/development.rb (hunk 3 above)
Comment (ag4ta) [reply config/environments/development.rb]: I've reverted this line :)

**Pull request merged**